realm/realm-library/proguard-rules-base.pro

```diff
@@ -0,0 +1,2 @@
+# It's OK no to exist SyncObjectServerFacade in base library.
```

Comment:
updated.
I meant that there is a dynamic reference to `SyncObjectServerFacade` in base library, but the absence `SyncObjectServerFacade` in base library is normal.